JS/modules/date&time.js

```diff
@@ -0,0 +1,14 @@
+const dateLine = document.querySelector('.dayDate');
+const navBar = document.querySelector('nav');
+const displayDate = () => {
```

Comment:
- [ ] For Date and time, kindly consider using `luxon` library instead of `Date`. The Package should be added as a .js file to the project (in full or minified version)